database/index.js

```diff
@@ -19,6 +19,18 @@ const getListings = (callback) => {
   });
 };
 
+const getBooking = (id, callback) => {
+  const numId = Number(id);
+  connec.query(`SELECT * from bookings WHERE listing_id = ${numId}`, (err, results) => {
+    if (err) {
+      callback(err);
+    } else {
+      callback(null, results);
```

Comment:
it's a personal opinion, but I have hard time guessing what does this `null` part do in here. It might be more readable if we can manage to just send the `results` here and the `callback` function can determine how to handle that on its own.  That way, the code is going to be consistent here.